ibm/data_source_ibm_is_subnet.go

```diff
@@ -238,6 +238,10 @@ func subnetGetByNameOrID(d *schema.ResourceData, meta interface{}) error {
 				break
 			}
 		}
+		if subnet == nil {
```

Comment:
Can you change the error message something like below

return fmt.Errorf("No subnet with name [%s]", name)